webpack/ForemanTasks/Components/common/ActionButtons/ActionButton.js

```diff
@@ -1,27 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { CancelButton } from './CancelButton';
-import { ResumeButton } from './ResumeButton';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { ActionButtons } from 'foremanReact/components/common/ActionButtons/ActionButtons';
 
 export const ActionButton = ({ id, name, availableActions, taskActions }) => {
   const isResume = availableActions.resumable;
+  const isCancel = availableActions.cancellable;
```

Comment:
instead of re-declaring these boolean variables,
you can destruct `availableActions` at line 6:
```js
export const ActionButton = ({ id, name, availableActions: { resumable, cancellable }, taskActions }) => {
```

Reply:
Changed, thanks!